cypress/integration/orders/orders.CreateOrderLineForPEMix.spec.js

```diff
@@ -5,18 +5,22 @@ import TopMenu from '../../support/fragments/topMenu';
 import SearchHelper from '../../support/fragments/finance/financeHelper';
 import interactorsTools from '../../support/utils/interactorsTools';
 import OrderLines from '../../support/fragments/orders/orderLines';
+import newOrganization from '../../support/fragments/organizations/newOrganization';
 
 describe('orders: Test PO search', () => {
   const order = { ...NewOrder.defaultOrder };
+  const organization = { ...newOrganization.defaultUiOrganizations };
   let orderNumber;
 
   before(() => {
     cy.getAdminToken();
-    cy.getOrganizationApi({ query: 'name="Amazon.com"' })
-      .then(organization => {
-        order.vendor = organization.id;
+
+    cy.createOrganizationApi(organization);
+    cy.getOrganizationApi({ query: `name="${organization.name}"` })
```

Comment:
add the same api method into related fragment + mark existing cy.method as deprecated with link to new fragment method